src/greyVest/ExpandableTable.js

```diff
@@ -135,16 +136,43 @@ let ExpandableTable = inject(TableState)(
                 <th
                   key={`${c.field}${i}`}
                   {...(c.enableSort && {
-                    onClick: () => columnSort(c),
-                    style: { cursor: 'pointer', textDecoration: 'underline' },
+                    style: { cursor: 'pointer' },
                   })}
                 >
-                  {F.callOrReturn(_.getOr(F.autoLabel(c.field), 'label', c))}
-                  {c.enableSort && c.field === sortField && sortDir
-                    ? sortDir === 'asc'
-                      ? '▲'
-                      : '▼'
-                    : ''}
+                  <span>
+                    {F.callOrReturn(_.getOr(F.autoLabel(c.field), 'label', c))}
+                    {c.enableSort && (
+                      <Popover
+                        trigger={<Icon icon="TableColumnMenu" />}
+                        position={`bottom ${
+                          i === columns.length - 1 ? 'right' : 'center'
+                        }`}
+                        style={{
+                          userSelect: 'none',
+                          width: 'auto',
+                        }}
+                      >
+                        <DropdownItem
+                          onClick={() => {
+                            c.sortDir = 'asc'
+                            columnSort(c)
```

Comment:
this would be cleaner to just call `columnSort` with a fresh object instead of mutating `c` first (and it would be less code since you can make it one line)